frontend/src/app/offers/offers.model.ts

```diff
@@ -0,0 +1,11 @@
+export class Offer {
```

Comment:
When I was refactoring `UserService` into `AuthService` I create `User` and used `interface` instead of `class` @Bosmanfrx what do You think?

It also looks more like TypeScript decalration, and there is a custom to put that kind of declarations in `d.ts` files. Maybe we should merge this file with previously created `user.d.ts` and crate `types.d.ts`?

Reply:
I agree it should be an interface. We won't be creating these as objects, we are just using it for types.

Reply:
Guys, interface works only during compile time. According to Angular styleguide it's recommended to use Class over Interface for the typing. The main difference is that class will persist when compiled while Interface are purely removed since they do not serve any usage. I recommend to leave it as Class

Reply:
Ok, you convinced me. There is conflict between Typescript and Angular guideline. But I think Angular guideline should take precedence right now.